### PR TITLE
tests: Add unit tests for expected values

### PR DESCRIPTION
**Pull request by bayotop**

### CODE REVIEW

File: src/sources/audio.test.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ describe('Sources', () => {
     it('returns expected value type depending on the browser', async () => {
       const result = getAudioFingerprint()
 
-      if (isSafari() && isMobile() && (getBrowserMajorVersion() ?? 0) < 12) {
+      if (isUnsupportedBrowser()) {
         // WebKit has stopped telling its real version in the user-agent string since version 605.1.15,
         // therefore the browser version has to be checked instead of the engine version.
```
Comment (Finesse) [inline src/sources/audio.test.ts:11]: This comment should be inside the `isUnsupportedBrowser` function

File: src/sources/audio.test.ts [hunk 2]
```diff
@@ -21,5 +21,25 @@ describe('Sources', () => {
...
+      if (isUnsupportedBrowser()) {
+        return
+      } else {
+        if (typeof result !== 'function') {
+          throw new Error('Expected to be a function')
+        }
+
+        const first = await result()
+        const second = await result()
+        expect(second).toBe(first)
```
Comment (Finesse) [inline src/sources/audio.test.ts:37]: All calls of the same `result` function return literally the same promise, so this assertion can't fail. Therefore `getAudioFingerprint` should be called twice instead.

**Review by Finesse — CHANGES_REQUESTED** [review]
Good job, thank you. Just a couple of minor issues.

File: src/sources/audio.test.ts (hunk 2 above)
Comment (bayotop) [reply src/sources/audio.test.ts]: Good catch, updated. Thanks!

**Review by Finesse — APPROVED** [review]

**Review by Finesse — APPROVED** [review]

**Pull request merged**